regions/core/pixcoord.py

```diff
@@ -11,7 +11,7 @@
 class PixCoord(object):
     """Pixel coordinates.
 
-    This class can represent scalar or array pixel coordinates.
+    This class can represent scalar or array of pixel coordinates.
```

Comment:
Very minor comment: remove "of" again, I think the previous sentence was OK. If you add "of" I think it would have to be "arrays of".